spec/requests/api/v1/targets/destroy_spec.rb

```diff
@@ -0,0 +1,67 @@
+RSpec.describe 'DELETE api/v1/targets', type: :request do
+  let(:user) { create(:user) }
+  let!(:target) { create(:target, user: user) }
+
+  let(:headers) { auth_headers }
+
+  subject { delete api_v1_target_path(target.id), headers: headers }
+
+  context 'when the request is correct' do
+    it 'returns a successful response' do
+      subject
+
+      expect(response).to be_successful
+    end
+
+    it 'decreases the targets count' do
+      expect {
+        subject
+      }.to change(Target, :count).by(-1)
+    end
+  end
+
+  context 'when the target does not exist' do
+    let!(:non_existent_id) { target.id + 1 }
```

Comment:
```suggestion
    let(:non_existent_id) { target.id + 1 }
```
it shouldn't be needed here